app/controllers/api/v1/tasks_controller.rb

```diff
@@ -39,6 +39,15 @@ def update
       else
         render_error_message(@task)
       end
+
+    when 'finalized'
+      if @task.is_finalizable?(current_user)
+        @task.update(status: params[:activity], user: current_user)
+        render json: { message: 'You finalized the delivery' }
```

Comment:
Perhaps: 'Your delivery is complete and you made someone happy :) Thanks for being a Helping Hander!"

Reply:
finalized comes from the requester right?

Reply:
So it should probably be: We are happy that you received your order. Please be in touch if you have any further questions.

Reply:
We gonna change all of those messages in one PR